### ISSUE TITLE
Two enhancements to the Quantized Record Option.

### ISSUE THREAD

**Issue by ca3games** [LMMS/lmms#5981]
### Enhancement Summary
This is more like a new version of:
#5976
Seeing now LMMS has a quantized record option, I think my two suggestions could be very easily be implemented with it.
My initial idea would be basically two new behaviours and an optional one.

Basically:

1. Allow to make longer notes, basically the quantization length button will be the minimal unit of time being quantized, but you could sustain your notes being played with the keyboard and it will quantize to the nearest unit of the quantized button.
2. Allow to make the non ghost notes being selectable.
3. (Optional) Allow a % of the notes to be longer or being shorter to the quantization, basically will be like a semi real time, like the normal recording where time passes only when the keys are being pressed, this will allow for notes to be "offsync" and "humanized" and allow notes to be slurred between them.

#### Justification
These three improvements will make the quantized recording into something much much better, and I would say even better than professional tools recording and humanizing features.

#### Mockup
Basically each idea can easily be made:

1. No GUI change, only set the quantization button as the minimal note being quantized while being held.
2. A button next to the ghost note, similar to the alpha button in layers in Krita, or allow the ghost note being pressed multiple times to turn on and off a transparency checkboard background for the icon.
3. A % list next to the quantized button that consists of % like 5/10/20/25/50 options for the % of the humanizing off sync quantization.
